examples/eris/src/index.js

```diff
@@ -1,14 +1,32 @@
-const Eris = require('eris');
+const Eris = require("eris");
+const fs = require("fs");
 const bot = Eris(`Bot ${process.env.DISCORD_TOKEN}`);
+const prefix = process.env.PREFIX || "!";
```

Comment:
```suggestion
const prefix = process.env.PREFIX;
```

No need, already defaults to `!` when using the starter.

Reply:
No, but like if user wants his own prefix while using the starter without editing the code, this is a good thing then